### PR TITLE
Fixes #13

### LINKED ISSUE #13: Storing the Highest Score, not reset everytime
Objectives:
-> Store the Highest Score in Local Storage
-> Render the Highest Score from Local Storage everytime the game starts
-> Enhancing the Game Experience.

### PR DESCRIPTION
**Pull request by RhyshaKachari**
I have stored the highest score in local storage that renders whenever you open the game
Screenshot 2022-10-03 224716

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -10,17 +11,39 @@ if (!gameOn) {
   $(document).on("keypress", gameStarts);
 }
 
+window.onload = function highsc() {
+  console.log("heelo");
```
Comment (anurrags) [inline index.js]: remove this line

**Review by anurrags — COMMENTED** [review]
Check you code again, there is a bug. Whenever same button is to be blinked consecutively them no blinking effect of button take is seen. Try to play the game for sometime and you will notice the bug

File: index.js (hunk 1 above)
Comment (RhyshaKachari) [reply index.js]: Okay I am working on it.

**Comment by RhyshaKachari** [comment]
I have removed the line and checked the blinking of buttons too. It is working fine after my latest commit. Can you please review it once?

**Pull request merged**